### ISSUE TITLE
Multi-inputs with dynamic axes in ONNX Graph not corrected reported on loading

### ISSUE THREAD

**Issue by mfuntowicz** [triton-inference-server/server#2123]
**Description**
I've exported a `bert-base-cased` model from Hugging Face transformers which includes 3 inputs: 

- `input_ids`: [batch, sequence]
- `token_type_ids`: [batch, sequence]
- `attention_mask`: [batch, sequence]

Here, `batch` and `sequence` refers to dynamic axis exported through ONNX IR.
Netron correctly reports these axes: 

![image](https://user-images.githubusercontent.com/2241520/96040124-136a7c00-0e6a-11eb-8ab4-8e744d7412c4.png)

When loading the model in Triton with no `config.pbtxt` implicitly requesting Triton to generate such model configuration, it reports only one dynamic axis:

```json
{
    "name": "bert-base-cased",
    "platform": "onnxruntime_onnx",
    "backend": "onnxruntime",
    "version_policy": {
        "latest": {
            "num_versions": 1
        }
    },
    "max_batch_size": 1,
    "input": [
        {
            "name": "token_type_ids",
            "data_type": "TYPE_INT64",
            "dims": [
                -1
            ]
        },
        {
            "name": "input_ids",
            "data_type": "TYPE_INT64",
            "dims": [
                -1
            ]
        },
        {
            "name": "attention_mask",
            "data_type": "TYPE_INT64",
            "dims": [
                -1
            ]
        }
    ],
    "output": [
        {
            "name": "output_1",
            "data_type": "TYPE_FP32",
            "dims": [
                768
            ]
        },
        {
            "name": "output_0",
            "data_type": "TYPE_FP32",
            "dims": [
                -1,
                768
            ]
        }
    ],
    "batch_input": [],
    "batch_output": [],
    "optimization": {
        "priority": "PRIORITY_DEFAULT",
        "input_pinned_memory": {
            "enable": true
        },
        "output_pinned_memory": {
            "enable": true
        }
    },
    "instance_group": [
        {
            "name": "bert-base-cased",
            "kind": "KIND_GPU",
            "count": 1,
            "gpus": [
                0
            ],
            "profile": []
        }
    ],
    "default_model_filename": "model.onnx",
    "cc_model_filenames": {},
    "metric_tags": {},
    "parameters": {},
    "model_warmup": []
}
```

**Triton Information**
20.09-py3

**Are you using the Triton container or did you build it yourself?**

Docker version from Nvidia repository

**To Reproduce**

Export the model from hugging face transformers through the following: 

```bash
python <transformers_path/src/transformers/convert_graph_to_onnx.py --model bert-base-cased --framework pt --opset 12 bert-base-cased.onnx
```

**Expected behavior**

I would expect Triton to report `dims = [-1, -1]` for all the inputs / outputs, mapping all the dynamic axes serialized through ONNX IR.

**Comment by deadeyegoodwin** [comment]
Read https://github.com/triton-inference-server/server/blob/master/docs/model_configuration.md#inputs-and-outputs

The full tensor shape is determine by the dims and max_batch_size. When you auto-generate a configuration triton doesn't know what max-batch-size to use so it conservatively chooses "1". So the generated config means that triton will accept tensors with shape [ x, -1 ], where x is 1. If you set max-batch-size to, for example, 16. Shape will then be [ y, -1 ], where y must be 1<= y <= 16.
